khard/data/template.yaml

```diff
@@ -2,6 +2,10 @@
 # from the full name below if not given.
 Formatted name : 
 
+# kind (requires vcard >= 4.0)
```

Comment:
There is (to my knowledge) no version >4 and we do not know if and how such a version would support a kind attribute.

```suggestion
# kind (requires vcard 4.0)
```